pkg/pillar/zedcloud/send.go

```diff
@@ -127,6 +127,12 @@ func SendOnAllIntf(ctx *ZedCloudContext, url string, reqlen int64, b *bytes.Buff
 					url, reqlen, resp.StatusCode)
 				return resp, nil, remoteTemporaryFailure, err
 			}
+			//http.StatusForbidden is a special case i.e. even if bailOnHTTPErr is not set
+			//we bail out, since caller needs to be notified immediately for triggering any
+			//reaction based on this
```

Comment:
Comment would make more sense on line 76 after we discuss the bail.

Reply:
Done